src/main/scala/cromwell/engine/db/slick/SlickDataAccess.scala

```diff
@@ -0,0 +1,451 @@
+  }
+
+  /**
+   * Creates a row in each of the backend-info specific tables for each call in `calls` corresponding to the backend
+   * `backend`.  Or perhaps defer this?
+   */
+  override def createWorkflow(workflowInfo: WorkflowInfo,
+                              workflowInputs: Traversable[SymbolStoreEntry],
+                              calls: Traversable[Call],
+                              backend: Backend): Future[Unit] = {
+
+    val action = for {
+
+      workflowExecutionInsert <- dataAccess.workflowExecutionsAutoInc +=
+        new WorkflowExecution(
+          workflowInfo.workflowId.toString,
+          WorkflowSubmitted.toString,
+          new Date().toTimestamp)
+
+      _ <- dataAccess.workflowExecutionAuxesAutoInc += new WorkflowExecutionAux(
+        workflowExecutionInsert.workflowExecutionId.get,
+        workflowInfo.wdlSource,
+        workflowInfo.wdlJson)
+
+      symbolInsert <- dataAccess.symbolsAutoInc ++=
+        workflowInputs.toSeq map { symbol =>
+          new Symbol(
+            workflowExecutionInsert.workflowExecutionId.get,
+            symbol.key.scope,
+            symbol.key.name,
+            symbol.key.iteration,
+            if (symbol.isInput) IoInput else IoOutput,
+            symbol.wdlType.toWdlString,
+            symbol.wdlValue.map(wdlValueToString).orNull)
+        }
+
+      // NOTE: Don't use DBIO.seq for **transforming** sequences
+      // - DBIO.seq(mySeq: _*) runs *any* items in sequence, but converts Seq[ DBIOAction[_] ] to DBIOAction[ Unit ]
+      // - DBIO.sequence(mySeq) converts Seq[ DBIOAction[R] ] to DBIOAction[ Seq[R] ]
+      // - DBIO.fold(mySeq, init) converts Seq[ DBIOAction[R] ] to DBIOAction[R]
+
+      _ <- DBIO.sequence(calls.toSeq map {
+        call => for {
+          executionInsert <- dataAccess.executionsAutoInc +=
+            new Execution(
+              workflowExecutionInsert.workflowExecutionId.get,
+              call.taskFqn,
+              ExecutionStatus.NotStarted.toString)
+
+          _ <- backend match {
+            case _: LocalBackend =>
+              dataAccess.localJobsAutoInc +=
+                new LocalJob(
+                  executionInsert.executionId.get,
+                  -1,
+                  -1)
+            case null =>
+              throw new NotImplementedError("Backend is null")
+            case unknown =>
+              throw new NotImplementedError("Unknown backend: " + backend.getClass)
+          }
+        } yield ()
+      })
+
+    } yield ()
+
+    runTransaction(action)
+  }
+
+  override def getWorkflowState(workflowId: WorkflowId): Future[Option[WorkflowState]] = {
+
+    val action = for {
+
+      workflowExecutionResultOption <- dataAccess.workflowExecutions.filter(
+        _.workflowExecutionUuid === workflowId.toString).result.headOption
+
+      workflowState = workflowExecutionResultOption map { workflowExecutionResult =>
+        WorkflowState.fromString(workflowExecutionResult.status)
```

Comment:
It seems we use the term "status" everywhere except for workflows; I think this should become `WorkflowStatus` for consistency with the database and calls.